op-node/flags/flags.go

```diff
@@ -33,6 +33,11 @@ var (
 		Usage:  "Rollup chain parameters",
 		EnvVar: prefixEnvVar("ROLLUP_CONFIG"),
 	}
+	Network = cli.StringFlag{
+		Name:   "network",
+		Usage:  "Predefined network selection",
```

Comment:
We should list which possible choices there are here in the `Usage` field, it may get stale but its better ux than not knowing at all